tests/Fixtures/Integration/misc/PHP8_2.test

```diff
@@ -0,0 +1,179 @@
+--TEST--
+PHP 8.2 test.
+--RULESET--
+{
+    "@PhpCsFixer": true,
+    "@PHP82Migration": true
+}
+--REQUIREMENTS--
+{"php": 80200}
+--EXPECT--
+<?php
+
+// https://wiki.php.net/rfc/readonly_classes
+final readonly class Foo
```

Comment:
Should `final` be before or after `readonly`? Or should it be configurable somehow?

Reply:
It will be covered by the upcoming PER CS release: https://github.com/php-fig/per-coding-style/blob/9dc210da72285afd5ab10ab0bd34659ff5b0529f/spec.md#46-modifier-keywords.

Reply:
would be a nice new fixer :)

Reply:
is now part of https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6624 , please review if you've some time :)